modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/styles/components/_drop-down-filter.scss

```diff
@@ -0,0 +1,4 @@
+.drop-down-menu-items {
+	padding: 0 !important;
+	max-width: 200px !important;
```

Comment:
Is this `!important` required?

Reply:
the padding one was removed, but the max-width is needed to replace the component's default width.